app.py

```diff
@@ -1,18 +1,15 @@
 import os
-from langchain.llms import OpenAI
-from langchain.embeddings import OpenAIEmbeddings
-import streamlit as st
-
-from langchain.document_loaders import PyPDFLoader
-from langchain.vectorstores import Chroma
 
+import streamlit as st
 from langchain.agents.agent_toolkits import (
     create_vectorstore_agent,
     VectorStoreToolkit,
     VectorStoreInfo
 )
-
-os.environ['OPENAI_API_KEY'] = 'input api key here'

```

Comment:
This one should also not be needed. We can easily set it like `OPENAI_API_KEY=<YOUR_API_KEY> streamlit run app.py`